Utils/Elasticsearch/requests/hashtag-steps-statistics-q.json

```diff
@@ -11,6 +11,8 @@ GET <index>/task/_search
     "bool": {
       "must": [
         {"term": {"hashtag_list": %%HASHTAG_LOWERCASE%%}},
+        {"exists": {"field": "start_time"}},
+        {"exists": {"field": "end_time"}},
```

Comment:
This condition will affect all the aggregations. Why should not we add value of "input_events" of a task with undefined start/end time to the total sum of input events number (for example)?